### PR TITLE
feat: replace ReactQuill component with plain Quill

### PR DESCRIPTION
**Pull request by frdomovic**
# feat: replace ReactQuill component with plain Quill

## Summary:
As using ReactQuill has limitations and a lot of bugs in nextJS, I create [new issue](https://github.com/zenoamaro/react-quill/issues/936) in their repo and until that is fixed I did not find another way to make some of the wanted behaviours work.

Replaced with plain Quill which uses refs to manipulate inputs and keyDown changes.
Connected Apps [PR](https://github.com/calimero-is-near/PortalApps/pull/439)

Fixes : [DEV-159](https://calimero.atlassian.net/browse/DEV-159)

## Tests:

https://github.com/calimero-is-near/VM/assets/93442516/b1c1189b-bae3-4955-b580-d2be882aa53b

### CODE REVIEW

**Comment by MatejVukosav** [comment]
https://github.com/zenoamaro/react-quill/issues/936 is closed?

**Comment by frdomovic** [comment]
> [zenoamaro/react-quill#936](https://github.com/zenoamaro/react-quill/issues/936) is closed?

He tagged me in another comment saying they have fix for it and not to open new issues, decided in the end to ditch react-quill as the new implementation offers much more flexibility regarding features

File: src/lib/components/MarkdownEditor.js [hunk 1]
```diff
@@ -1,27 +1,77 @@
...
+    e.preventDefault();
+    const clipboardData = e.clipboardData;
+    let pastedData;
+
+    if (clipboardData.types.includes('text/html')) {
+      pastedData = clipboardData.getData('text/html');
+    } else if (clipboardData.types.includes('text/plain')) {
+      pastedData = clipboardData.getData('text/plain');
+    }
+    const processedData = sanitizePasteHtml(pastedData);
```
Comment (MatejVukosav) [inline src/lib/components/MarkdownEditor.js:29]: do you need to sanitize content if type is text/plain?

**Review by MatejVukosav — APPROVED** [review]

File: src/lib/components/MarkdownEditor.js (hunk 1 above)
Comment (frdomovic) [reply src/lib/components/MarkdownEditor.js:29]: Yes

**Pull request merged**